[2022-06-16 11:57:41.518] [main] [trace] [File Watcher (node.js)] Request to start watching: /home/ghaffari/Documents/project/BigFan/backend/~/root/User (excludes: <none>, includes: <all>),/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json (excludes: <none>, includes: <all>)
[2022-06-16 11:57:41.518] [main] [debug] Starting VS Code
[2022-06-16 11:57:41.518] [main] [debug] from: /snap/code/99/usr/share/code/resources/app
[2022-06-16 11:57:41.518] [main] [debug] args: {"_":["/home/ghaffari/Documents/project/BigFan/backend"],"diff":false,"add":false,"goto":false,"new-window":false,"reuse-window":false,"wait":false,"user-data-dir":"~/root","help":false,"list-extensions":false,"show-versions":false,"pre-release":false,"version":false,"verbose":true,"status":false,"prof-startup":false,"no-cached-data":false,"prof-v8-extensions":false,"disable-extensions":false,"disable-gpu":false,"ms-enable-electron-run-as-node":false,"telemetry":false,"debugRenderer":false,"enable-smoke-test-driver":false,"logExtensionHostCommunication":false,"skip-release-notes":false,"skip-welcome":false,"disable-telemetry":false,"disable-updates":false,"disable-keytar":false,"disable-workspace-trust":false,"disable-crash-reporter":false,"crash-reporter-id":"56cfae92-0e03-4eab-88ee-c5ab491d3bd9","skip-add-to-recently-opened":false,"unity-launch":false,"open-url":false,"file-write":false,"file-chmod":false,"force":false,"do-not-sync":false,"trace":false,"force-user-env":false,"force-disable-user-env":false,"open-devtools":false,"__sandbox":false,"__enable-file-policy":false,"no-proxy-server":false,"no-sandbox":false,"nolazy":false,"force-renderer-accessibility":false,"ignore-certificate-errors":false,"allow-insecure-localhost":false,"disable-dev-shm-usage":false,"logsPath":"/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741"}
[2022-06-16 11:57:41.518] [main] [trace] Resolving machine identifier...
[2022-06-16 11:57:41.518] [main] [trace] Resolved machine identifier: 47f9b99844903c556f29bcc04e7dc8dcc74229c3270a2424c794fd8e8162c874
[2022-06-16 11:57:41.518] [main] [trace] Main->SharedProcess#connect
[2022-06-16 11:57:41.520] [main] [trace] [File Watcher (node.js)] Started watching: '/home/ghaffari/Documents/project/BigFan/backend/~/root/User'
[2022-06-16 11:57:41.522] [main] [trace] [File Watcher (node.js)] Error: ENOENT: no such file or directory, stat '/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json'
[2022-06-16 11:57:41.591] [main] [trace] StorageMainService: creating global storage
[2022-06-16 11:57:41.593] [main] [info] update#setState idle
[2022-06-16 11:57:41.594] [main] [trace] lifecycle (main): phase changed (value: 2)
[2022-06-16 11:57:41.595] [main] [trace] windowsManager#open
[2022-06-16 11:57:41.596] [main] [trace] windowsManager#open pathsToOpen [{"workspace":{"id":"756958d73ea15e0e9530f0d54e6bed2b","uri":{"$mid":1,"fsPath":"/home/ghaffari/Documents/project/BigFan/backend","path":"/home/ghaffari/Documents/project/BigFan/backend","scheme":"file"}},"type":2,"exists":true}]
[2022-06-16 11:57:41.597] [main] [trace] windowsManager#doOpenFolderOrWorkspace {"folderOrWorkspace":{"workspace":{"id":"756958d73ea15e0e9530f0d54e6bed2b","uri":{"$mid":1,"fsPath":"/home/ghaffari/Documents/project/BigFan/backend","external":"file:///home/ghaffari/Documents/project/BigFan/backend","path":"/home/ghaffari/Documents/project/BigFan/backend","scheme":"file"}},"type":2,"exists":true}}
[2022-06-16 11:57:41.597] [main] [trace] IPC Object URL: Registered new channel vscode:a619d830-25bb-4bf2-8556-85fbf6260866.
[2022-06-16 11:57:41.598] [main] [trace] window#validateWindowState: validating window state on 2 display(s) {"mode":1,"x":2095,"y":296,"width":1024,"height":768}
[2022-06-16 11:57:41.598] [main] [trace] window#validateWindowState: multi-monitor working area {"x":1920,"y":0,"width":1920,"height":1200}
[2022-06-16 11:57:41.598] [main] [trace] window#ctor: using window state {"mode":1,"x":2095,"y":296,"width":1024,"height":768}
[2022-06-16 11:57:41.674] [main] [trace] window#load: attempt to load window (id: 1)
[2022-06-16 11:57:41.683] [main] [trace] StorageMainService: creating workspace storage (756958d73ea15e0e9530f0d54e6bed2b)
[2022-06-16 11:57:41.684] [main] [trace] windowsManager#open used window count 1 (workspacesToOpen: 0, foldersToOpen: 1, emptyToRestore: 0, emptyToOpen: 0)
[2022-06-16 11:57:41.684] [main] [trace] lifecycle (main): phase changed (value: 3)
[2022-06-16 11:57:41.684] [main] [trace] resolveShellEnv(): skipped (VSCODE_CLI is set)
[2022-06-16 11:57:41.807] [main] [trace] resolveShellEnv(): skipped (VSCODE_CLI is set)
[2022-06-16 11:57:42.301] [main] [trace] [Disk FileSystemProvider]: atomic read operation started (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:42.301] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - request to acquire resource lock (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:42.301] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - new resource lock created (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:42.302] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock dispose() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:42.302] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock removed from resource-lock map (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:42.302] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock barrier open() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:42.431] [main] [trace] [Disk FileSystemProvider]: atomic read operation started (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/keybindings.json)
[2022-06-16 11:57:42.431] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - request to acquire resource lock (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/keybindings.json)
[2022-06-16 11:57:42.431] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - new resource lock created (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/keybindings.json)
[2022-06-16 11:57:42.433] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock dispose() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/keybindings.json)
[2022-06-16 11:57:42.433] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock removed from resource-lock map (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/keybindings.json)
[2022-06-16 11:57:42.433] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock barrier open() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/keybindings.json)
[2022-06-16 11:57:42.501] [main] [info] Starting extension host with pid 13083 (fork() took 7 ms).
[2022-06-16 11:57:42.859] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - request to acquire resource lock (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/tasks.log)
[2022-06-16 11:57:42.860] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - new resource lock created (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/tasks.log)
[2022-06-16 11:57:42.860] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - request to acquire resource lock (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/extensions.log)
[2022-06-16 11:57:42.860] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - new resource lock created (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/extensions.log)
[2022-06-16 11:57:42.860] [main] [trace] [Disk FileSystemProvider]: open() - storing lock for handle 82 (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/tasks.log)
[2022-06-16 11:57:42.860] [main] [trace] [Disk FileSystemProvider]: open() - storing lock for handle 85 (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/extensions.log)
[2022-06-16 11:57:42.874] [main] [trace] [Disk FileSystemProvider]: close() - resource lock removed from handle-lock map 85
[2022-06-16 11:57:42.874] [main] [trace] [Disk FileSystemProvider]: close() - disposing lock for handle 85
[2022-06-16 11:57:42.874] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock dispose() (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/extensions.log)
[2022-06-16 11:57:42.874] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock removed from resource-lock map (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/extensions.log)
[2022-06-16 11:57:42.874] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock barrier open() (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/extensions.log)
[2022-06-16 11:57:42.874] [main] [trace] [Disk FileSystemProvider]: close() - resource lock removed from handle-lock map 82
[2022-06-16 11:57:42.874] [main] [trace] [Disk FileSystemProvider]: close() - disposing lock for handle 82
[2022-06-16 11:57:42.874] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock dispose() (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/tasks.log)
[2022-06-16 11:57:42.874] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock removed from resource-lock map (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/tasks.log)
[2022-06-16 11:57:42.874] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock barrier open() (/home/ghaffari/Documents/project/BigFan/backend/~/root/logs/20220616T115741/output_1_20220616T115742/tasks.log)
[2022-06-16 11:57:44.283] [main] [trace] SharedProcess: on vscode:createSharedProcessMessageChannel
[2022-06-16 11:57:44.283] [main] [trace] IPC Object URL: Registered new channel vscode:5c00204e-83f2-4625-9c41-013e4e119d09.
[2022-06-16 11:57:44.394] [main] [trace] resolveShellEnv(): skipped (VSCODE_CLI is set)
[2022-06-16 11:57:44.511] [main] [trace] SharedProcess: IPC ready
[2022-06-16 11:57:44.523] [main] [trace] Main->SharedProcess#connect: connection established
[2022-06-16 11:57:44.530] [main] [trace] [Disk FileSystemProvider]: atomic read operation started (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:44.530] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - request to acquire resource lock (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:44.530] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - new resource lock created (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:44.531] [main] [trace] [File Watcher (node.js)] Request to start watching: /home/ghaffari/Documents/project/BigFan/backend/~/root/User (excludes: <none>, includes: <all>),/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json (excludes: <none>, includes: <all>)
[2022-06-16 11:57:44.531] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock dispose() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:44.531] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock removed from resource-lock map (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:44.531] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock barrier open() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:44.534] [main] [trace] [File Watcher (node.js)] Started watching: '/home/ghaffari/Documents/project/BigFan/backend/~/root/User'
[2022-06-16 11:57:44.534] [main] [trace] [File Watcher (node.js)] Error: ENOENT: no such file or directory, stat '/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json'
[2022-06-16 11:57:44.568] [main] [trace] SharedProcess: Overall ready
[2022-06-16 11:57:45.017] [main] [trace] SharedProcess: onWorkerConnection {"process":{"moduleId":"vs/platform/files/node/watcher/watcherMain","type":"fileWatcher"},"reply":{"windowId":1,"channel":"vscode:createSharedProcessWorkerMessageChannelResult","nonce":"f1f9a48e-b6ce-4fc3-83e9-233c875e8493"}}
[2022-06-16 11:57:47.337] [main] [trace] [Disk FileSystemProvider]: atomic read operation started (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:47.337] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - request to acquire resource lock (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:47.337] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - new resource lock created (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:47.338] [main] [trace] [Disk FileSystemProvider]: atomic read operation started (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/tasks.json)
[2022-06-16 11:57:47.338] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - request to acquire resource lock (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/tasks.json)
[2022-06-16 11:57:47.338] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - new resource lock created (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/tasks.json)
[2022-06-16 11:57:47.338] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock dispose() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:47.338] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock removed from resource-lock map (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:47.338] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock barrier open() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/settings.json)
[2022-06-16 11:57:47.339] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock dispose() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/tasks.json)
[2022-06-16 11:57:47.339] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock removed from resource-lock map (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/tasks.json)
[2022-06-16 11:57:47.339] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock barrier open() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/tasks.json)
[2022-06-16 11:58:11.610] [main] [trace] update#checkForUpdates, state =  idle
[2022-06-16 11:58:11.611] [main] [info] update#setState checking for updates
[2022-06-16 11:58:11.613] [main] [info] update#setState idle
[2022-06-16 11:58:35.495] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - request to acquire resource lock (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/globalStorage/storage.json)
[2022-06-16 11:58:35.495] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - new resource lock created (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/globalStorage/storage.json)
[2022-06-16 11:58:35.496] [main] [trace] [Disk FileSystemProvider]: open() - storing lock for handle 94 (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/globalStorage/storage.json)
[2022-06-16 11:58:35.533] [main] [trace] [Disk FileSystemProvider]: close() - resource lock removed from handle-lock map 94
[2022-06-16 11:58:35.534] [main] [trace] [Disk FileSystemProvider]: close() - disposing lock for handle 94
[2022-06-16 11:58:35.534] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock dispose() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/globalStorage/storage.json)
[2022-06-16 11:58:35.534] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock removed from resource-lock map (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/globalStorage/storage.json)
[2022-06-16 11:58:35.534] [main] [trace] [Disk FileSystemProvider]: createResourceLock() - resource lock barrier open() (/home/ghaffari/Documents/project/BigFan/backend/~/root/User/globalStorage/storage.json)
[2022-06-16 12:00:30.614] [main] [trace] Lifecycle#window.on('close') - window ID 1
[2022-06-16 12:00:30.615] [main] [trace] Lifecycle#onBeforeCloseWindow.fire() - window ID 1
[2022-06-16 12:00:30.616] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.616] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.log (node:internal/console/constructor:359:26)
    at m.trace (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:2563)
    at w.trace (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5087)
    at /snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:33:9910
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
[2022-06-16 12:00:30.618] [main] [trace] IPC Object URL: Removed channel vscode:a619d830-25bb-4bf2-8556-85fbf6260866.
[2022-06-16 12:00:30.618] [main] [trace] Lifecycle#window.on('closed') - window ID 1
[2022-06-16 12:00:30.618] [main] [trace] Lifecycle#onWillShutdown.fire()
[2022-06-16 12:00:30.619] [main] [trace] IPC Object URL: Removed channel vscode:5c00204e-83f2-4625-9c41-013e4e119d09.
[2022-06-16 12:00:30.620] [main] [trace] storageMainService#onWillShutdown()
[2022-06-16 12:00:30.620] [main] [info] Waiting for extension host with pid 13083 to exit.
[2022-06-16 12:00:30.620] [main] [trace] SharedProcess: disposing worker (reason: 'closed') {"process":{"moduleId":"vs/platform/files/node/watcher/watcherMain","type":"fileWatcher"},"reply":{"windowId":1,"channel":"vscode:createSharedProcessWorkerMessageChannelResult","nonce":"f1f9a48e-b6ce-4fc3-83e9-233c875e8493"}}
[2022-06-16 12:00:30.621] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.621] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.log (node:internal/console/constructor:359:26)
    at m.trace (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:2563)
    at w.trace (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5087)
    at Object.dispose (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:32:15925)
    at d (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:14:4972)
    at p.clear (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:14:5491)
    at p.dispose (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:14:5429)
    at re.dispose (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:14:6030)
    at re.dispose (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:5279)
    at BrowserWindow.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:74:73159)
    at BrowserWindow.emit (node:events:402:35)
[2022-06-16 12:00:30.622] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.622] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.622] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.622] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.623] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.623] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.623] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.623] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.624] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.624] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.624] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.624] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.625] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.625] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.625] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.625] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.625] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.625] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.626] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.626] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.626] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.626] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.626] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.626] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.627] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.627] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.627] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.627] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.627] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.627] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.628] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.628] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.628] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.628] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.629] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.629] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.629] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.629] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.629] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.629] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.630] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.630] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.630] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.630] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.631] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.631] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.631] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.631] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.632] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.632] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.632] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.632] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.632] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.632] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.633] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.633] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.633] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.633] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.634] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.634] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.634] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.634] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.634] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.634] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.635] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.635] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.635] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.635] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
[2022-06-16 12:00:30.635] [main] [error] [uncaught exception in main]: Error: write EPIPE
[2022-06-16 12:00:30.635] [main] [error] Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:164:15)
    at writeGeneric (node:internal/stream_base_commons:155:3)
    at Socket._writeGeneric (node:net:795:11)
    at Socket._write (node:net:807:8)
    at writeOrBuffer (node:internal/streams/writable:389:12)
    at _write (node:internal/streams/writable:330:10)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at console.value (node:internal/console/constructor:285:16)
    at console.warn (node:internal/console/constructor:364:26)
    at m.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:3116)
    at w.error (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:31:5331)
    at yt.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:61955)
    at S.unexpectedErrorHandler (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59228)
    at S.onUnexpectedError (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:354)
    at R (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:5:615)
    at process.<anonymous> (/snap/code/99/usr/share/code/resources/app/out/vs/code/electron-main/main.js:75:59309)
    at process.emit (node:events:402:35)
    at process._fatalException (node:internal/process/execution:170:25)
